OpenRA.Game/GameRules/Ruleset.cs

```diff
@@ -45,7 +45,23 @@ public class Ruleset
 
 			foreach (var a in Actors.Values)
 				foreach (var t in a.TraitInfos<IRulesetLoaded>())
-					t.RulesetLoaded(this, a);
+					try {
```

Comment:
Please add braces to the outer loops, and then move these ones on to their own line to fit with our usual style conventions. (and the same again below).